plugins/coverimage.koplugin/main.lua

```diff
@@ -189,58 +662,20 @@ function CoverImage:addToMainMenu(menu_items)
             },
             -- menu entry: filename dialog
             {
-                text = _("Set image path"),
+                text = _("Set image"),
```

Comment:
It would be helpful to display the value (i.e. the full path) right next to the menu item — this goes for the fallback image too. So, perhaps something like?

```suggestion
                text_func = function()
                    return T(_("Set image: %1"), G_reader_settings:readSetting("cover_image_path", self.cover_image_path))
                end,
```

Reply:
I have put it in the help text (long press ) as the path can be quite long.